src/ui/public/modals/index.js

```diff
@@ -0,0 +1,3 @@
+import './confirm_modal';
+import './safe_confirm';
```

Comment:
Having safe_confirm in modals seems weird. The safe_confirm is still its own service, the fact that is uses a modal now is an implementation detail...